hazenlib/ghosting.py

```diff
@@ -223,25 +244,28 @@ def get_ghosting(dcm, plotting=False) -> dict:
         img = cv.rectangle(img.copy(), (x1, y1), (x2, y2), (255, 0, 0), 1)
 
         ax.imshow(img)
+        fig.savefig(f'{report_path}.png')
         return fig, ghosting
 
     return None, ghosting
 
 
-def main(data: list, report_path=False) -> dict:
+def main(data: list, report_path = False) -> dict:
 
     results = {}
-    # figures = []
+    #figures = []
     for dcm in data:
         try:
             key = f"{dcm.SeriesDescription.replace(' ', '_')}_{dcm.EchoTime}ms_NSA-{dcm.NumberOfAverages}"
+            if report_path:
+               report_path = key
         except AttributeError as e:
             print(e)
             key = f"{dcm.SeriesDescription}_{dcm.SeriesNumber}"
         try:
-            fig, results[key] = get_ghosting(dcm, report_path)
-            if report_path:
-                fig.savefig(key + '.png')
+            fig, results[key] = get_ghosting(dcm, report_path=True)
+
+
```

Comment:
I don't follow the changes here? Possibly linked to my previous comment about the image `True.png` being output every time?

Reply:
Previously I could not get the script to output an image, so that's why I changed it, but now I need to fix this problem of always outputting an image that I created. I'll fix this.